src/Glpi/Application/Environment.php

```diff
@@ -231,12 +250,22 @@ public function shouldEnableExtraDevAndDebugTools(): bool
         };
     }
 
```

Comment:
To prevent BC break.
```suggestion
    public function shouldAddExtraE2EDataDuringInstallation(): bool
    {
        return $this->shouldAddExtraCypressDataDuringInstallation() || $this->shouldAddExtraPlaywrightDataDuringInstallation();
    }

```